starboard/android/shared/video_render_algorithm.cc

```diff
@@ -26,7 +26,7 @@ namespace shared {
 
 namespace {
 
-const SbTimeMonotonic kBufferTooLateThreshold = -32 * kSbTimeMillisecond;
+const SbTimeMonotonic kBufferTooLateThreshold = -16 * kSbTimeMillisecond;
```

Comment:
This is also used for non-tunneled playback, and is probably carefully tuned.  What's the implications for non-tunneled playback if it's reduced?

Reply:
I think 32ms is a too large tolerance for dropped frames. But as it's carefully tuned, I changed it back to 32ms.